src/geo/transform.js

```diff
@@ -257,6 +262,14 @@ class Transform {
         this._calcMatrices();
     }
 
+    get averageElevation(): number {
+        return this._averageElevation;
+    }
+    set averageElevation(averageElevation: number) {
+        this._averageElevation = averageElevation;
+        this._calcMatrices();
```

Comment:
dont think this is necessary, _averageElevation is something transform calculates internally during a fixed point in the frame. Recalculating matrices is only necessary if we intend to ovveride it and get the matrices back in the same frame which we dont need rn.

Reply:
I'd like to remove it, but I find that it's completely broken and doesn't update it without it.